tests/btrfs-progs/run.pm

```diff
@@ -58,19 +57,20 @@ sub get_test_list {
     return @tests;
 }
 
-# Run a single test and return test result
+# Run a single test, return test result and copy log to file
 # test - test to run(e.g. 001)
 sub test_run {
     my $num = shift;
     script_run("./clean-tests.sh");
-    my $ret    = script_output("TEST=$num\\* ./$CATEGORY-tests.sh", 600, proceed_on_failure => 1);
+    my $ret    = script_output("TEST=$num\\* ./" . CATEGORY . '-tests.sh', 600, proceed_on_failure => 1);
     my $status = 'PASSED';
     if ($ret =~ /[Ff]ailed/) {
         $status = 'FAILED';
     }
     elsif ($ret =~ /NOTRUN/) {
         $status = 'SKIPPED';
     }
+    script_run('cp ' . CATEGORY . '-tests-results.txt ' . LOG_DIR . CATEGORY . "/$num.txt");
```

Comment:
`script_run` does not evaluate the return code. Can you use `assert_script_run` instead here? Otherwise I suggest to explicitly check for the return code of `script_run`

Reply:
Its just copy test log to logs folder, I need test keep running even copy failed.

Reply:
Thanks, I added subtitle with  btrfs-progs